test/benchmark.py

```diff
@@ -39,7 +39,7 @@ def urllib_get(url_list):
     assert url_list
     for url in url_list:
         now = time.time()
-        r = urllib.urlopen(url)
```

Comment:
these test/ changes are unnecessary as they are excluded.  looking at them again, these changes are probably not worth doing as part of this changeset.